pkg/rehearse/jobs.go

```diff
@@ -396,6 +396,64 @@ func AddRandomJobsForChangedTemplates(templates []config.ConfigMapSource, toBeRe
 	return rehearsals
 }
 
+func getPresubmitForRegistryStep(step config.RegistryStep, configs config.ByFilename, prConfigPresubmits map[string][]prowconfig.Presubmit) (string, *prowconfig.Presubmit) {
+	for _, ciopConfig := range configs {
```

Comment:
You get random access to the elements of a map so this function will return a random output every time -- we want it to be deterministic so that when you /retest it picks the same one